.github/actions/fetch-releases-json/action.yaml

```
@@ -0,0 +1,41 @@
+name: Fetch Releases JSON
+description: Fetches the _releases.json file to the current repository.
+
+inputs:
+  image-name:
+    description: 'OCI image name to fetch the releases JSON for'
+    required: true
+  releases-branch:
+    description: 'Branch where the _releases.json file is located'
+    required: false
+    default: '_releases'
+  directory:
+    description: 'Directory where the _releases branch of the OCI Factory is checked out'
+    default: oci-factory-releases
+
+
+
+runs:
+  using: "composite"
+  steps:
+    - name: Fetch _releases.json
+      uses: actions/checkout@v4
+      with:
+        path: ${{ inputs.directory }}
+        ref: ${{ inputs.releases-branch }}
+
+    - name: Copy _releases.json
+      shell: bash
+      run: |
+        if [[ "$RUNNER_DEBUG" == "1" ]]; then
+          set -x
+        fi
+
+        if [[ "${{ inputs.image-name}}" = "*" ]]; then
+          echo "Copying all _releases.json files"
+          cd ${{ inputs.directory }}
+          find . -name "_releases.json" -exec cp --parents {} ../
```

```suggestion
          find . -name "_releases.json" -exec cp --parents {} $OLDPWD \;
```

Cause you can't guarantee the `inputs.directory` is just one level

---

Nice catch!

---

@zhijie-yang for the future, please consider applying the change requests as suggested, if appropriate, cause in this case the line wasn't properly fixed, and it's now ending up in errors: https://github.com/canonical/oci-factory/actions/runs/17420437102